engine/src/conversion/analysis/fun/mod.rs

```diff
@@ -1196,6 +1198,16 @@ impl<'a> FnAnalyzer<'a> {
         }
         let mut cxxbridge_name = make_ident(&cxxbridge_name);
 
+        let friendly_name = {
```

Comment:
I appreciate that much of the naming stuff is a bit of a mess, but please do your best to be consistent here with the way other configuration directives are approached.

In `byvalue_checker.rs` we have:

```rust
        let pod_requests = config
            .get_pod_requests()
            .iter()
            .map(|ty| QualifiedName::new_from_cpp_name(ty))
            .collect();
```

and the comment in `new_from_cpp_name` suggests this is what you should be using.